website/docs/compute/customizing-functions/fully-qualified-function-names.md

```diff
@@ -28,7 +34,7 @@ To execute a function in Compute you will use a URL representation of FQFN. To r
 /{environment}.{userid}/{namespace}/{function}/{version}
 
 Example:
-/com.awesomeco.nawronuq98hqwekj198fkljbeco/default/record-event/v1.0.0
+/dev.suborbital.nawronuq98hqwekj198fkljbeco/default/record-event/v1.0.0
```

Comment:
Should this be `com.example` to match earlier prose?

Reply:
Yep!